### PR TITLE
Update Body Text/Heading - Rancher backup-restore-operator Default Storage Location

### PR DESCRIPTION
**Pull request by sunilarjun**
## Description

Updating the default storage location body text to better describe options for configuring the location in the [Rancher How-To](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery#default-storage-location-configuration) section, and revising the heading/term for the Rancher backup-restore-operator Helm chart in both the How-To and Reference pages to add a bit of clarification.

Storage location configuration reference page: https://ranchermanager.docs.rancher.com/reference-guides/backup-restore-configuration/storage-configuration

Related to and fixes https://github.com/rancher/rancher-docs/issues/1037.

### CODE REVIEW

File: docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md [hunk 1]
```diff
@@ -91,10 +91,10 @@ A migration is performed by following [these steps.](migrate-rancher-to-new-clus
 
 ## Default Storage Location Configuration
 
-Configure a storage location where all backups are saved by default. You will have the option to override this with each backup, but will be limited to using an S3-compatible or Minio object store.
+Configure a storage location where all backups are saved by default. Various configuration options are available, including specifying an S3-compatible object store as the location for individual backups or choosing an existing `StorageClass` during installation of the `backup-restore-operator` helm chart. You will also have the option to override the configured storage location with each backup, but will be limited to using an S3-compatible or Minio object store.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md:94]:
```suggestion
Configure a default storage location for your backups. There are various configuration options, such as specifying an S3-compatible object store as the location for individual backups or choosing an existing `StorageClass` during installation of the `backup-restore-operator` Helm chart. You also have the option to override the configured storage location with each backup, but are limited to using an S3-compatible or Minio object store.
```

Avoiding passive voice and future tense when not necessary

File: versioned_docs/version-2.7/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md [hunk 2]
```diff
@@ -91,10 +91,10 @@ A migration is performed by following [these steps.](migrate-rancher-to-new-clus
 
 ## Default Storage Location Configuration
 
-Configure a storage location where all backups are saved by default. You will have the option to override this with each backup, but will be limited to using an S3-compatible or Minio object store.
+Configure a storage location where all backups are saved by default. Various configuration options are available, including specifying an S3-compatible object store as the location for individual backups or choosing an existing `StorageClass` during installation of the `backup-restore-operator` helm chart. You will also have the option to override the configured storage location with each backup, but will be limited to using an S3-compatible or Minio object store.
```
Comment (martyav) [inline versioned_docs/version-2.7/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md:94]:
```suggestion
Configure a default storage location for your backups. There are various configuration options, such as specifying an S3-compatible object store as the location for individual backups or choosing an existing `StorageClass` during installation of the `backup-restore-operator` Helm chart. You also have the option to override the configured storage location with each backup, but are limited to using an S3-compatible or Minio object store.
```

File: versioned_docs/version-2.8/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md [hunk 3]
```diff
@@ -91,10 +91,10 @@ A migration is performed by following [these steps.](migrate-rancher-to-new-clus
 
 ## Default Storage Location Configuration
 
-Configure a storage location where all backups are saved by default. You will have the option to override this with each backup, but will be limited to using an S3-compatible or Minio object store.
+Configure a storage location where all backups are saved by default. Various configuration options are available, including specifying an S3-compatible object store as the location for individual backups or choosing an existing `StorageClass` during installation of the `backup-restore-operator` helm chart. You will also have the option to override the configured storage location with each backup, but will be limited to using an S3-compatible or Minio object store.
```
Comment (martyav) [inline versioned_docs/version-2.8/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md:94]:
```suggestion
Configure a default storage location for your backups. There are various configuration options, such as specifying an S3-compatible object store as the location for individual backups or choosing an existing `StorageClass` during installation of the `backup-restore-operator` Helm chart. You also have the option to override the configured storage location with each backup, but are limited to using an S3-compatible or Minio object store.
```

**Review by martyav — CHANGES_REQUESTED** [review]
I made some suggestions regarding passive voice and future tense. Otherwise looks good.

File: docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md (hunk 1 above)
Comment (sunilarjun) [reply docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md]: Thanks for the review Marty! Pushed the changes to all versions and will make a note to maintain present tense in the future.

File: docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md [hunk 4]
```diff
@@ -91,10 +91,10 @@ A migration is performed by following [these steps.](migrate-rancher-to-new-clus
...
 
 For information on configuring these options, refer to [this page.](../../../reference-guides/backup-restore-configuration/storage-configuration.md)
 
-### Example values.yaml for the rancher-backup Helm Chart
+### Example YAML File: Rancher Backup Helm Chart
 
-The example [values.yaml file](../../../reference-guides/backup-restore-configuration/storage-configuration.md#example-valuesyaml-for-the-rancher-backup-helm-chart) can be used to configure the `rancher-backup` operator when the Helm CLI is used to install it.
+The example [values.yaml file](../../../reference-guides/backup-restore-configuration/storage-configuration.md#example-valuesyaml-for-the-rancher-backup-helm-chart) can be used to configure the `backup-restore-operator` when the Helm CLI is used to install it.
```
Comment (btat) [inline docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md:100]:
```suggestion
The example [values.yaml file](../../../reference-guides/backup-restore-configuration/storage-configuration.md#example-yaml-file-rancher-backup-helm-chart) can be used to configure the `backup-restore-operator` when the Helm CLI is used to install it.
```

File: versioned_docs/version-2.7/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md [hunk 5]
```diff
@@ -91,10 +91,10 @@ A migration is performed by following [these steps.](migrate-rancher-to-new-clus
...
 
 For information on configuring these options, refer to [this page.](../../../reference-guides/backup-restore-configuration/storage-configuration.md)
 
-### Example values.yaml for the rancher-backup Helm Chart
+### Example YAML File: Rancher Backup Helm Chart
 
-The example [values.yaml file](../../../reference-guides/backup-restore-configuration/storage-configuration.md#example-valuesyaml-for-the-rancher-backup-helm-chart) can be used to configure the `rancher-backup` operator when the Helm CLI is used to install it.
+The example [values.yaml file](../../../reference-guides/backup-restore-configuration/storage-configuration.md#example-valuesyaml-for-the-rancher-backup-helm-chart) can be used to configure the `backup-restore-operator` when the Helm CLI is used to install it.
```
Comment (btat) [inline versioned_docs/version-2.7/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md:100]:
```suggestion
The example [values.yaml file](../../../reference-guides/backup-restore-configuration/storage-configuration.md#example-yaml-file-rancher-backup-helm-chart) can be used to configure the `backup-restore-operator` when the Helm CLI is used to install it.
```

File: versioned_docs/version-2.8/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md [hunk 6]
```diff
@@ -91,10 +91,10 @@ A migration is performed by following [these steps.](migrate-rancher-to-new-clus
...
 
 For information on configuring these options, refer to [this page.](../../../reference-guides/backup-restore-configuration/storage-configuration.md)
 
-### Example values.yaml for the rancher-backup Helm Chart
+### Example YAML File: Rancher Backup Helm Chart
 
-The example [values.yaml file](../../../reference-guides/backup-restore-configuration/storage-configuration.md#example-valuesyaml-for-the-rancher-backup-helm-chart) can be used to configure the `rancher-backup` operator when the Helm CLI is used to install it.
+The example [values.yaml file](../../../reference-guides/backup-restore-configuration/storage-configuration.md#example-valuesyaml-for-the-rancher-backup-helm-chart) can be used to configure the `backup-restore-operator` when the Helm CLI is used to install it.
```
Comment (btat) [inline versioned_docs/version-2.8/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md:100]:
```suggestion
The example [values.yaml file](../../../reference-guides/backup-restore-configuration/storage-configuration.md#example-yaml-file-rancher-backup-helm-chart) can be used to configure the `backup-restore-operator` when the Helm CLI is used to install it.
```

**Review by btat — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2.8/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md (hunk 6 above)
Comment (sunilarjun) [reply versioned_docs/version-2.8/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md]: Thanks for the catch Billy! Just updated across the versions.

**Review by martyav — APPROVED** [review]

**Review by btat — APPROVED** [review]

**Pull request merged**